support-frontend/assets/pages/digital-subscription-checkout/components/orderSummary/orderSummaryStyles.js

```diff
@@ -1,21 +1,32 @@
 import { css } from '@emotion/core';
 import { headline, body, textSans } from '@guardian/src-foundations/typography/obj';
 import { space } from '@guardian/src-foundations';
-import { background, brandAltBackground, text } from '@guardian/src-foundations/palette';
+import { background, brandAltBackground, text, border } from '@guardian/src-foundations/palette';
 import { from, between, until } from '@guardian/src-foundations/mq';
 
 export const wrapper = css`
   background-color: ${background.primary};
   color: ${text.primary};
-  padding-bottom: ${space[3]}px;
+  ${until.desktop} {
+    padding: ${space[3]}px;
+  }
+  ${until.tablet} {
+    box-shadow: 0px 4px 4px ${border.secondary};
+  }
 `;
 
 export const topLine = css`
   display: flex;
   justify-content: space-between;
   width: calc(100%-${space[3]}px * 2);
-  margin: ${space[3]}px;
   align-items: center;
+  padding: ${space[3]}px;
+
+  ${until.desktop} {
+    border-top: 1px solid #EDEDED;
```

Comment:
Is `#EDEDED` in '@guardian/src-foundations/palette'?

Reply:
Good point - I have updated this.